### ISSUE TITLE
Issue with import

### ISSUE THREAD

**Issue by cobookman** [jprante/elasticsearch-jdbc#37]
WHy is it that this line for SQL works:
       .... "sql" : "SELECT \"class\" as \"_index\", ID as \"_id\", Course as \"course.name\",  Name as \"prof.name\", SUBSTRING_INDEX(Course,\" \", -1) as \"department.name\", GPA as \"gpa.name\" FROM Spring2012" ...

But something like this does not:
       ...."sql" : "SELECT \"class\" as \"_index\", ID as \"_id\", Course as \"course.name\",  Name as \"course.prof\", SUBSTRING_INDEX(Course,\" \", -1) as \"course.department\", GPA as \"course.gpa\" FROM Spring2012" ...

I've made sure that the table has no 'null' or empty values. Also GPA and Profs have different values for the same ID.

Tldr having a structure of:
     { course.name, prof.name, department.name, gpa.name } 
is ok where as:
     { course : { name, prof, department, gpa } }
is not.

Thanks
